app/jobs/application_job.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 class ApplicationJob < ActiveJob::Base
+  around_perform do |_, block|
```

Comment:
I still think it would be better to set the default strategy for Chewy globally than to set it here.

Reply:
Add a comment linking to this discussion. It’s good to have this documented.